sandbox/gas/tube/go

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python3.5
```

Comment:
Bump the version because SOLVCON has migrated to python 3. (and 3.5 specifically so far for [the python11 compatibility issue](https://github.com/solvcon/solvcon/commit/2329189d383988c5ea0cb5705cfb69fc5e4c4eda))